x-pack/packages/kbn-cloud-security-posture-common/types/latest.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema, TypeOf } from '@kbn/config-schema';
```

Comment:
For the first iteration it might work, but eventually I think the source of truth for the versioned schema should be the common package, as we are using types and schemas in both the plugin and in the package